changelog.d/20250416_123930_nlevesq_docker_compose_prod_override.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Create a changelog entry for every new user-facing change. Please respect the following instructions:
+- Indicate breaking changes by prepending an explosion 💥 character.
+- Prefix your changes with either [Bugfix], [Improvement], [Feature], [Security], [Deprecation].
+- You may optionally append "(by @<author>)" at the end of the line, where "<author>" is either one (just one)
+  of your GitHub username, real name or affiliated organization. These affiliations will be displayed in
+  the release notes for every release.
+-->
+
+<!-- - 💥[Feature] Foobarize the blorginator. This breaks plugins by renaming the `FOO_DO` filter to `BAR_DO`. (by @regisb) -->
+<!-- - [Improvement] This is a non-breaking change. Life is good. (by @billgates) -->
+
+- [Improvement] Add support for `docker-compose.prod.override.yml` when using
+  `tutor local` commands
```

Comment:
- please add your github username at the end of changelog entry.

Reply:
Note that adding the username in the changelog entry is optional. If OP does not need public recognition, that's fine :-)

Reply:
@rhysyngsun Hi, are you good with merging this without mentioning your GitHub in the changelog entry? You can update the changelog if you want to. Thanks

Reply:
You can merge it as-is.